src/ExtCmd.actor.cpp

```diff
@@ -697,23 +697,22 @@ struct BuildInfoCmd {
 	static Future<Reference<ExtMsgReply>> call(Reference<ExtConnection> ec,
 	                                           Reference<ExtMsgQuery> query,
 	                                           Reference<ExtMsgReply> reply) {
-		reply->addDocument(BSON("version"
-		                        << EXT_SERVER_VERSION
-		                        << "gitVersion"
-		                        << "<string>"
-		                        << "OpenSSLVersion"
-		                        << ""
-		                        << "sysInfo"
-		                        << "<string>"
-		                        << "loaderFlags"
-		                        << "<string>"
-		                        << "compilerFlags"
-		                        << "<string>"
-		                        << "allocator"
-		                        << "<string>"
-		                        << "versionArray" << BSON_ARRAY(2 << 4 << 10) << "javascriptEngine"
-		                        << "<string>"
-		                        << "bits" << 64 << "debug" << false << "maxBsonObjectSize" << 16777216 << "ok" << 1.0));
+		reply->addDocument(BSON("version" << EXT_SERVER_VERSION << "gitVersion"
```

Comment:
If you are changing the format of BSON builder in the code, its probably better to do it as key-value pairs. Its more readable like that.

Sample is [here](https://github.com/FoundationDB/fdb-document-layer/blob/master/src/ExtCmd.actor.cpp#L303).

It's for future reference not a blocker for this PR.

Reply:
Sure thing.